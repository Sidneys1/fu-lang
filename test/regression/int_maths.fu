main: u32() = {
  a: u32 = 100;
  b: u32 = 2;
  c: u32 = a - b; // 98
  d: u32 = a * b; // 200
  e: u32 = a / b; // 50
  return c + d + e; // 98 + 200 + 50 = 348
};
