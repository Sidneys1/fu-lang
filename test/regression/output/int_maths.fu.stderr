INFO:fu.compiler:Output binary `a.out`: 855 Bytes
